src/server/api/debug.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+
+import logger from "../logging";
+import c from "cookie";
+
+export default function debugRouter() {
+    const api = express.Router();
+
+    logger.info("************ Adding Debug handlers **********");
+
+    logger.info("adding the GET /apps/categories handler");
```

Comment:
Might be good to remove this line to avoid confusion

Reply:
Oops! I just removed that line.